augur/normalized_payout.py

```diff
@@ -15,6 +15,6 @@ def is_invalid(self):
   def payout(self):
     '''Payout Set for the Dispute Crowdsourcer.
 
-    Returns list<int or str>
+    Returns list[int or decimal.Decimal]
```

Comment:
When is it an ```int``` and when is it a ```decimal.Decimal```?

Reply:
The Payout Set will hold the payout distribution for each outcome once the market has been finalized. There is a calculation performed to set each value, and they will be prices so it would make sense to make this a `list[decimal.Decimal` instead.

Reply:
Agreed.